### PR TITLE
Blog: Sleep function in Javascript

### PR DESCRIPTION
**Pull request by tsl143**

### CODE REVIEW

File: content/blog/sleep-function-in-javascript/index.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
+title: 'Sleep function in Javascript'
+date: "2020-02-16"
+description: A simple sleep function in Javscript to halt he execution of Javascript for small time
```
Comment (championshuttler) [inline content/blog/sleep-function-in-javascript/index.md]:
```suggestion
description: A simple sleep function in Javscript to halt the execution of Javascript for small time
```

**Review by championshuttler — DISMISSED** [review]
LGTM 💯

File: content/blog/sleep-function-in-javascript/index.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+---
+title: 'Sleep function in Javascript'
+date: "2020-02-16"
+description: A simple sleep function in Javscript to halt the execution of Javascript for small time
```
Comment (tsl143) [inline content/blog/sleep-function-in-javascript/index.md]:
```suggestion
description: A simple sleep function in Javscript to halt the execution of Javascript for small period
```

File: content/blog/sleep-function-in-javascript/index.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+description: A simple sleep function in Javscript to halt the execution of Javascript for small period
+tags:
+  - javascript
+show: true
+author: trishul
+---
+
+There are times while writing javascript, when we might want to halt the execution of Javascript for small period say 2 seconds. May be while writing test cases, or avoiding race condition, etc. Javascript provides (natively) no way to do this, but we can write our own `sleep` function.
+
+To do this we will use `Promise` / `await`, `setTimeout` (let's just face it no one wants to use callbacks anymore 🤐).
```
Comment (tsl143) [inline content/blog/sleep-function-in-javascript/index.md]:
```suggestion
To do this we will use `Promise` / `await`, `setTimeout` (let's just face it no one wants to use `callbacks` anymore 🤐).
```

**Pull request merged**